### PR TITLE
[teamsyncd]: Add LAG members to syncd when created.

### PR DESCRIPTION
**Pull request by marian-pritsak**
It is possible that LAG members are created after teamd
creates LAG. In that case we have to monitor RTM_NEWLINK
messages and addd them via teamdctl.

Signed-off-by: marian-pritsak <marianp@mellanox.com>

### CODE REVIEW

**Comment by stcheng** [comment]
@marian-pritsak could you update the pull request as i just merged #182.
And according to my understanding of this pull request, we don't need to have teamdctl in /etc/network/interfaces file, correct?

File: teamsyncd/teamsync.cpp [hunk 1]
```diff
@@ -28,18 +29,27 @@ void TeamSync::onMsg(int nlmsg_type, struct nl_object *obj)
     if ((nlmsg_type != RTM_NEWLINK) && (nlmsg_type != RTM_DELLINK))
         return;
 
-    string lagName = rtnl_link_get_name(link);
+    string intfName = rtnl_link_get_name(link);
```
Comment (stcheng) [inline teamsyncd/teamsync.cpp]: or change to teamName?

File: teamsyncd/teamsync.cpp [hunk 2]
```diff
@@ -207,3 +236,24 @@ void TeamSync::TeamPortSync::readMe()
 {
     team_handle_events(m_team);
 }
+
+void TeamSync::TeamPortSync::syncMember(const std::string& intfName)
+{
+    const std::string config(teamdctl_config_get_raw(m_tdc));
+    const std::string intfNameStr("\"" + (intfName + "\""));
```
Comment (stcheng) [inline teamsyncd/teamsync.cpp]: why there is one more '(' and ')' inside?

File: teamsyncd/teamsync.cpp (hunk 1 above)
Comment (marian-pritsak) [inline teamsyncd/teamsync.cpp]: Here we can have either a LAG or its member, so I picked a more general name

Comment (nikos-li) [reply teamsyncd/teamsync.cpp]: Please put a comment to that effect.

File: teamsyncd/teamsync.cpp [hunk 3]
```diff
@@ -121,6 +131,19 @@ TeamSync::TeamPortSync::TeamPortSync(const string &lagName, int ifindex,
...
     }
 
+    m_tdc = teamdctl_alloc();
+
+    if (m_tdc == NULL)
+    {
+        SWSS_LOG_THROW("Failed to allocate teamdctl context.");
+    }
+
+    err = teamdctl_connect(m_tdc, m_lagName.c_str(), NULL, NULL);
```
Comment (nikos-li) [inline teamsyncd/teamsync.cpp]: Where is this defined? I'm sorry but I couldn't find the function. Can you please tell me which file has it?

**Comment by marian-pritsak** [comment]
Not valid anymore, closing

**Pull request closed without merging**